readme.md

```diff
@@ -20,4 +20,12 @@ This repository is licensed under the Apache 2.0 License, except for the [server
 
 ## Development
 
+To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.
+
 This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.
+
+## Support
+
+- For Mattermost customers - Please open a support case.
+- For Questions and help, visit the [Jira Plugin channel](https://community.mattermost.com/core/channels/jira-plugin) on our Community server.
+- For suggestions and feedback, you can [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues) or post in the Jira Plugin channel.
```

Comment:
Should we explicitly point people who report bugs to the issues and everyone else to the channel? The line between question and feedback can be thin.

Reply:
I kind of agree with @hanzei.  We should direct questions, suggestions, and help, to visit the mattermost community server jira plugin channel. 
To report a bug, use github issues.  

I'm pre-approving because I am out this week. Thanks for the PR and fixes, @justinegeffen

Reply:
```suggestion
- To report a bug, please [open an issue](https://github.com/mattermost/mattermost-plugin-jira/issues).
```